lib/jellyfish/room.ex

```diff
@@ -117,6 +119,8 @@ defmodule Jellyfish.Room do
   @impl true
   def init([id, max_peers, video_codec]) do
     state = new(id, max_peers, video_codec)
+    Phoenix.PubSub.subscribe(Jellyfish.PubSub, id)
+    Phoenix.PubSub.broadcast(Jellyfish.PubSub, id, {:room_created, Node.self()})
```

Comment:
I think we should check whether broadcasting succeeds.
```suggestion
    :ok = Phoenix.PubSub.broadcast(Jellyfish.PubSub, id, {:room_created, Node.self()})
```